src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -180,55 +176,42 @@
                        view-finder (assoc layout :height (:width layout))]
                    (reset! qr-view-finder view-finder)))}])
 
-(defn- border
-  [border1 border2 corner]
-  [rn/view {:style (style/border border1 border2 corner)}])
+(defn- white-border
+  [corner]
+  (let [border-styles (style/white-border corner)]
+    [rn/view
+     [rn/view {:style (border-styles :border)}]
```

Comment:
I don't think this is right ..

Reply:
The function call to border-styles I mean.

Reply:
@ibrkhalil 
I am answering this and the previous comment.

Hash maps in Clojure are also functions, which receive a key and will return the value associated.

`({:a 1} :a)` => `1`

Like keywords being functions that extract itself from a map.
`(:a {:a 1})` => `1`

Or, like sets returning the element if it exists:
`(#{:a} :a)` => `:a`

(Actually, sets are hashmaps with the same key and value)

Reply:
Yeah, I keep forgetting
The function call to hashmaps will always remain confusing to me :)
Cool